Adds bridge IP and cidr for docker network

b3n0b1 opened the pull request:
## [DEV-20195](https://curalate.atlassian.net/browse/DEV-20195)

#### What does this PR do?
This PR adds the bip and cidr ip range options to the internal fork of the `chef-amazon-ecs-agent` cookbook.  This will allow us to specify the internal network and container host bridge IP to use for centralizing services like dogstatsd

#### Screenshots
_(not applicable)_

#### Testing done before PR? 
Tested end to end build on new jenkins job

#### Testing planned in staging?
_(not applicable)_

#### Additional areas of concern?
None

chrisbenincasa commented:
🚢

richid commented:
Code-wise loos fine to me! Do you mind changing the commit message title to use the _DEV-20195 ..._ format and toss the PR description into the commit message body?

Commit:
DEV-20195: Adds bip and cidr IP to docker opts

This PR adds the bip and cidr ip range options to the internal fork of the `chef-amazon-ecs-agent` cookbook.  This will allow us to specify the internal network and container host bridge IP to use for centralizing services like dogstatsd

b3n0b1 commented:
Commit message amended
